ddtrace/contrib/django/utils.py

```diff
@@ -1,3 +1,6 @@
+import json
+from urllib import parse
```

Comment:
You can import it from `ddtrace.internal.compat` to ensure Python 2 and 3 compatibility

Reply:
Didn't knew about that, will take a look, thanks.

Reply:
Fixed, thanks!